.gitignore

```diff
@@ -144,3 +144,6 @@ gradle-app.setting
 /junit.log
 .idea/modules/*.iml
 *.tokens
+gradle
```

Comment:
No, I disagree. The Gradle Wrapper and the contents of `gradle` should be checked in.

See:
 * https://github.com/github/gitignore/blob/master/Gradle.gitignore
 * https://stackoverflow.com/questions/20348451

Reply:
Fixed!